fhirstore/search/formatter.py

```diff
@@ -0,0 +1,63 @@
+from fhirstore.search.url_parser import URL_Parser
+import elasticsearch
+import logging
+
+
+class Formatter:
```

Comment:
I think the methods of this class (like the other classes) should use return values instead of modifying internal attributes. That way we would use a single instance of the Formatter for multiple bundles. @Jasopaum @elsiehoffet-94 what's your opinion on that ?

Reply:
Agreed
Modifying internal attributes can be useful if you really need to keep some state between 2 method calls but I don't think it's the case here